libs/utils/android/workloads/jankbench.py

```diff
@@ -20,8 +20,8 @@
 import logging
 
 from subprocess import Popen, PIPE
-from android import Screen, System, Workload
 
+from android import Screen, System, Workload
```

Comment:
Actually this is not required... the Workload class is already imported by the android module's __init__... thus we should better use what this module does everywhere else.

Moreover, the code to load Workloads it's a bit picky... we are kind of abusing the python in such a way that adding a workload is "as simple as" placing a new file under the `worloads` folder.

Thus I would better avoid playing with these imports and keep exactly what we have for jankbench ;-)